src/components/NavigationBar/NavigationBar.svelte

```diff
@@ -0,0 +1,122 @@
+<script lang="ts">
+	import type { Menu } from '$models/menu';
+
+	import Banner from './Banner.svelte';
+	import MenuPane from './MenuPane.svelte';
+	import MenuList from './MenuList.svelte';
+	import SideMenuButton from './SideMenuButton.svelte';
+	import SideMenuPane from './SideMenuPane.svelte';
+	import SideMenuList from './SideMenuList.svelte';
+
+	import PoliticianIcon from '$components/icons/PoliticianIcon.svelte';
+	import LawIcon from '$components/icons/LawIcon.svelte';
+	import VoteIcon from '$components/icons/VoteIcon.svelte';
+	import WeVisIcon from '$components/icons/WeVisIcon.svelte';
+	import NavigationPane from './NavigationPane.svelte';
+	import SearchInput from './SearchInput.svelte';
+
+	//import logoLong from '/static/images/logo/pw-long-white.png';
+	//import logoShort from '/static/images/logo/pw-short-white.png';
+
+	export let searchActionLink = '/search';
+
+	export let menuList: Menu[];
```

Comment:
I suggest that the `menuList` can be a `const` defined in this component because we want everywhere on our website to have the same menu. Then we also don't need to provide the duplicated data 2 times, in both layout and story files.